### DISCUSSION TITLE
Antivirus issues?

### DISCUSSION THREAD

**Discussion by disturbedalive2011** [spectreconsole/spectre.console#736] [Q&A] [answered]
Shortly after, including the spectre.console NuGet package in my program, I attempted to run the program but was thwarted by antivirus software.

Is there something to be concerned with on this? 

Could someone give a good reason why this happens?

**Comment by phil-scott-78** [comment]
Well I can assure you that isn't an intended behavior, at least. You have any more details e.g. the AV, the reported threat, etc that we could use to track down why it got identified?

**Reply by disturbedalive2011** [reply]
Thank you for the quick response; I didn't assume it was intended behavior. It threw me off quite a bit; the AV that reported the threat is Microsoft's Windows Defender. I have included a screenshot below.

![WindowsDefender](https://user-images.githubusercontent.com/3533058/154689646-cee80039-cb0b-44f9-8197-66d93e325ea8.png)

**Reply by phil-scott-78** [reply]
Someone else ran into this and posted some more logs. Tracking it here - #745 You wouldn't happen to have named your project `SpectreTest`?

**Comment by patriksvensson** [comment, marked as answer]
Quoting @phil-scott-78 here from #745

> Noticed Spectre.A was updated in [Defender update version 1.359.809.0](https://www.microsoft.com/en-us/wdsi/definitions/antimalware-definition-release-notes?requestVersion=1.359.809.0). I'm not seeing the error anymore so I'm going to go ahead and close this
